### PR TITLE
remove finished from teardown

### PR DESCRIPTION
**Pull request by ph**
Ref https://github.com/elastic/logstash/issues/3813

### CODE REVIEW

**Comment by suyograo** [comment]
LGTM

**Comment by jsvd** [comment]
I know this is somewhat out of scope for this PR but can you also rename teardown to close?

**Comment by ph** [comment]
@jsvd Done, renamed #teardown -> #close

File: lib/logstash/outputs/pipe.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ def receive(event)
     close_stale_pipes
   end # def receive
 
-  def teardown
+  def close
     @logger.info("Teardown: closing pipes")
```
Comment (jsvd) [inline lib/logstash/outputs/pipe.rb]: can you also rename this?

**Comment by ph** [comment]
yes, rebased too.

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**